docs/conf.py

```diff
@@ -36,6 +36,7 @@
     "colon_fence",
     "substitution",
 ]
+myst_heading_anchors = 7
```

Comment:
The commit message should probably mention that the key detail is adding this option, which makes `.md#` links work. (Previously, they did not).

Reply:
Mentioned.